.github/workflows/basic.yml

```diff
@@ -0,0 +1,100 @@
+name: Basic Tests and Documentation
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: "Default test set"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:    ["ubuntu-20.04"]
+        cxx:   ["g++"]
+        cc:    ["gcc"]
+        ver:   ["9"]
+        py:    ["3.6"]
+        cmake: ["3.12.x"]
+        mpi:   ["ON"]
+    env:
+        CC:         ${{ matrix.cc }}-${{ matrix.ver }}
+        CXX:        ${{ matrix.cxx }}-${{ matrix.ver }}
+        PY:         python${{ matrix.py }}
+        PYTHONPATH: /usr/local/lib/python${{ matrix.py }}/site-packages/
```

Comment:
I think we've just established that Ubuntu's system-Python doesn't look in PYTHONPATH (site-packages). Apparently `actions/setup-python@v2` does look there, or we'd be seeing failing builds. Is it a good idea to rely on this working?

Reply:
Using `PYTHONPATH` is a major code smell. What is this used for? If there are packages missing they should be installed inside the environment set up by `setup/python` rather than being "sneak peeked" from the system install.

Reply:
Good you mention this. It is caused by arbor hardwiring `site-packages` into `CMakeLists.txt`; *but* Debian uses `dist-packages`. Subject to another PR, I guess.

Reply:
For the unit tests we don't `make install` on Travis. We just set PYTHONPATH to `arbor/build/python`.
I would be happy to reproduce that behavior (and have other, separate, tests that test pip at a later date)

Reply:
Sure, no problem.

Reply:
So if I understand correctly, rather than installing the built files to their location, you point Python to the build location? Could this be added in a comment with the justification?

Reply:
Sure. :+1: